builder/tools_builder.livecodescript

```diff
@@ -76,38 +76,38 @@ end toolsBuilderRunBundle
 
 ////////////////////////////////////////////////////////////////////////////////
 
+private command toolsBuilderFilterExternals pFolder, pPlatform
+   -- Enter the extension directory
+   set the defaultFolder to pFolder
+
+   local tFiles
+   -- Remove any iOS code if we're not building the OSX installer
+   if pPlatform is not "macosx" then
+      put the files into tFiles
+      filter lines of tFiles with "*.lcext"
+      
+      -- Don't remove lcext files ith Android externals in them
+      local tLCExt
+      repeat for each line tLCExt in tFiles
+         get shell(merge("unzip -l '[[ tLCExt ]]'"))
+         if "Android/External-" is not in it then
+            get shell(merge("rm -rf '[[ tLCExt ]]'"))
+         end if
```

Comment:
I think you should probably just be using `rm -f` here because the extension is expected to be a normal file.